framework/include/cppmicroservices/BundleContext.h

```diff
@@ -1069,13 +1069,34 @@ class US_Framework_EXPORT BundleContext
  * @remarks An install location is an absolute path to a shared library or executable file
  * which may contain several bundles, i. e. acts as a bundle library.
  *
- * @remarks If the bundleManifest is passed in, it is installed. When the bundle is Started, the
- * BundleManifest is NOT read from the file (as it has already been installed). In the event that
- * the injected bundle manifest does NOT match the manifest in the bundle's file, the behavior of
- * the system is undefined. That is, the content of the injected manifest and the manifest on disk
- * are expected to be the same and are not compared.
- *
- * TODO: Add documentation of bundleManifest schema here
+ * @remarks
+ *     If the bundleManifest is passed in, it is installed. When the bundle is Started, the
+ *     BundleManifest is NOT read from the file (as it has already been installed). In the event
+ *     that the injected bundle manifest does NOT match the manifest in the bundle's file, the
```

Comment:
not a fan of using the word 'injected' in this context. What about 'provided', since its an optional parameter.

Reply:
Done.